### PR TITLE
FHAC-1231 did modifications to resolve the fortify issues

### PR DESCRIPTION
**Pull request by balmeeki**
did modifications to resolve the fortify issues for files mentioned in FHC-1231

### CODE REVIEW

**Comment by christophermay07** [comment]
Seeing a lot of spacing issues, mostly around tabs instead of spaces.  Do you have one of the CONNECT IDE formatters imported?

Hello  Chris - My Eclipse has the Connect IDE Formatter... I have no clue why these changes are happening

File: Product/Production/Services/DirectCore/src/main/java/gov/hhs/fha/nhinc/mail/MailUtils.java [hunk 1]
```diff
@@ -47,146 +47,171 @@
...
+			transport.sendMessage(message, recipients);
+			logHeaders(message);
+		} catch (AssertionError e) {
+			LOG.error("Assertion Error while sending.", e);
+			throw e;
+		} finally {
+			if (transport != null) {
+				try {
+					transport.close();
+				} catch (Exception e) {
```
Comment (christophermay07) [inline Product/Production/Services/DirectCore/src/main/java/gov/hhs/fha/nhinc/mail/MailUtils.java]: Is 'java.lang.Exception' the actual type being thrown, or can this catch be made more granular?

Comment (balmeeki) [reply Product/Production/Services/DirectCore/src/main/java/gov/hhs/fha/nhinc/mail/MailUtils.java]: Acknowledged.. Making more granular

**Review by mhpnguyen — APPROVED** [review]
(Optional) Recommend to use parameterized logging. However, this is not part of ticket scope.

**Review by sjarral112 — COMMENTED** [review]
Endorsed to merged, checked for parametrized and other formatting. Looks good.

**Pull request merged**